ext/intl/dateformat/dateformat.stub.php

```diff
@@ -136,7 +136,7 @@ public function getTimeZone(): IntlTimeZone|false {}
      * @tentative-return-type
      * @alias datefmt_set_timezone
      */
-    public function setTimeZone($timezone): ?bool {} // TODO return true on success
```

Comment:
since this is an alias (see `datefmt_set_timezone()`) all the changes should apply to the function as well

Reply:
what do you mean ? I applied the return change to datefmt_set_timezone or do you mean something else ?

Reply:
the `datefmt_set_timezone()`function in the stubs has to have the same return type. And the change log should also mention the return value change for the function as well

Reply:
I suppose this line should also be changed:
https://github.com/php/php-src/blob/ff71c9e15962b8d3797205770b4a8979d75bbcce/ext/intl/php_intl.stub.php#L353-L354

Reply:
oh ok I did not see that stub. cheers.